### PR TITLE
Enable failure on cycles.

### PR DESCRIPTION
**Pull request by polina-c**
Fixes https://github.com/polina-c/layerlens/issues/4

### CODE REVIEW

**Comment by polina-c** [comment]
@martingeorgiu, would you mind to accept [invitation](https://github.com/polina-c/layerlens/invitations) and review?

**Comment by polina-c** [comment]
PR: https://github.com/polina-c/layerlens/pull/17

**Review by sigurdm — APPROVED** [review]

File: bin/layerlens.dart [hunk 1]
```diff
@@ -27,18 +29,50 @@ enum _Options {
 
 void main(List<String> args) async {
   final parser = ArgParser()
-    ..addOption(_Options.path.name, defaultsTo: '.')
+    ..addFlag(
+      _Options.usage.name,
```
Comment (martingeorgiu) [inline bin/layerlens.dart:33]: I'm unsure about this parameter. General convention is to use `--help` which should be available by `args` package by default. However, that throws me an following exception:

```
Unhandled exception:
FormatException: Could not find an option named "help".
```

**Comment by polina-c** [comment]
Published. Let me know if there are issues.

**Pull request merged**